pages/deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
+
+# abort on errors
+set -e
+
+npm install -g vuepress
+# build
+npm run docs:build
+
+# navigate into the build output directory
+cd ./gh-pages
+
+# if you are deploying to a custom domain
+# echo 'www.example.com' > CNAME
+
+git init && \
+git config --global user.email @users.noreply.github.com && \
+git config --global user.name "Git Hub Deploy Action" && \
+git add . 
+git commit -m 'deploy'
+
+git push -f git@github.com:${GH_REPOSITORY}.git master:gh-pages
```

Comment:
To me this is the key part which pushes new set of pages to the gh-pages branch. It uses the deploy key which I have configured and has push access to the repository.

Reply:
@northtyphoon if this PR merges then azure.github.io/acr should auto publish and maybe we can ignore paths whcih can be ignored.